Date: Thu, 21 Mar 2019 15:51:31 -0500
Subject: [PATCH] cb-multios : Build single service

This allows a single service to be build by supplying the CGC_SERVICE
ENV var.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a0ccda8a..b96a28cd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -221,7 +221,7 @@ if(LINUX)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -z execstack -z norelro")
 endif()
 
-file(GLOB challenge_binaries challenges/*)
+file(GLOB challenge_binaries challenges/$ENV{CGC_CHALLENGE})
 foreach(cb_path ${challenge_binaries})
     if(IS_DIRECTORY ${cb_path} AND EXISTS ${cb_path}/CMakeLists.txt)
         # Get filename
-- 
2.17.1

